Scripts/RemediationScripts/Remediate-RemoteDesktopAccess.ps1

```diff
@@ -0,0 +1,357 @@
+            
+            $rdpExtensions += $cloudServiceExtension | Where-Object { ($null -ne $_) -and ($_.Extension -like "*RDP*") }
+            
+            if(($rdpExtensions | Measure-Object).Count -gt 0)
+            {
+                $item =  New-Object psobject -Property @{  
+                    CloudServiceName = $_.Name                
+                    ResourceGroupName = $_.ResourceGroupName
+                    RDPExtensionDetails = $rdpExtensions
+                }
+                    
+                $cloudServiceWithEnabledRDPAccess += $item        
+            }
+            else {
+                $cloudServiceWithDisabledRDPAccess += $_ | Select-Object @{Expression={($resource.ResourceGroupName)};Label="ResourceGroupName"},@{Expression={$resource.Name};Label="CloudServiceName"}
+            }
+        }   
+    }
+    catch {
+        Write-Host "Error occured while checking config of cloud service(s). ErrorMessage [$($_)]" -ForegroundColor $([Constants]::MessageType.Error)
+        break
+    }
+    
+    $totalCloudServiceWithEnableRDPAccess = ($cloudServiceWithEnabledRDPAccess | Measure-Object).Count
+    $totalCloudServiceWithDisabledRDPAccess = ($cloudServiceWithDisabledRDPAccess | Measure-Object).Count
+    
+    Write-Host "Cloud service(s) with enabled RDP access: [$($totalCloudServiceWithEnableRDPAccess)]"
+    Write-Host "Cloud service(s) with disabled RDP access: [$($totalCloudServiceWithDisabledRDPAccess)]"
+    Write-Host "`t"
+
+    # Performing remediation on cloud service(s).
+    try
+    {
+        if ($totalCloudServiceWithEnableRDPAccess -gt 0)
+        {
+            # Creating the log file
+            Write-Host "Backing up config of cloud service(s) detail." -ForegroundColor $([Constants]::MessageType.Info)
+            $cloudServiceWithEnabledRDPAccess | ConvertTo-json | out-file "$($folderpath)\CloudServiceWithEnabledRDPAccess.json"  
+            Write-Host "Path: $($folderpath)\CloudServiceWithEnabledRDPAccess.json"     
+            
+            # Asking user to verify logs and select 'Y' to proceed
+            if(-not $Force)
+            {
+                Write-Host "Do you want to disable RDP access on above listed cloud service(s)?" -ForegroundColor Yellow -NoNewline
+                $UserInput = Read-Host -Prompt "(Y|N)"
+                if($UserInput -ne "Y")
+                {
+                    return;
+                }
+            }
+            
+            Write-Host "`t"
+            Write-Host "Disabling RDP access on [$($totalCloudServiceWithEnableRDPAccess)] cloud service(s)..."
+            try 
+            {
+                $cloudServiceWithEnabledRDPAccess | ForEach-Object {
+                    # Disabling RDP access at service level
+                    $output = $_.RDPExtensionDetails | Remove-AzureServiceRemoteDesktopExtension -ServiceName $_.CloudServiceName -ErrorAction SilentlyContinue
```

Comment:
Do you want to provide the `UninstallConfiguration` flag to this command?

Reply:
Not considering for now, we can take it as enhancement.